client/scss/components/_listing.scss

```diff
@@ -487,6 +487,60 @@ ul.listing {
     }
 }
 
+footer.bulk-actions-choices {
+    @include transition(bottom 0.5s ease 0.5s);
+
+    &.hidden {
+        bottom: -200px;
+    }
+
+    .bulk-actions-more {
+        &.is-open {
+            ul.u-toggle {
+                display: flex;
+            }
+        }
+
+        ul {
+            bottom: 50px;
+            left: 30px;
+            flex-direction: column;
+        }
+    }
+}
+
+.footer__container {
```

Comment:
This impacts the regular action bar on the edit form. I suggest you test the changes together. The best thing to do is namespace it in `.bulk-actions-choices`

![Screenshot 2021-07-14 at 16 51 12](https://user-images.githubusercontent.com/31622/125652800-934cb92d-dfe2-42bf-b10c-ec405c935f56.png)